### ISSUE TITLE
Bug? / Enh ? : Led does not flash after pulling notification curtain

### ISSUE THREAD

**Issue by Som30ne** [quarck/CalendarNotification#265]
Scenario:
Define event with reminder.
Let reminder fire - see led flashing
Turn screen on.
Pull down notification curtain
** do NOT touch anything else
Turn screen off.
LED does not flash any more.

* I think that in the past, it used to continue flashing after turning screen off.

**Comment by quarck** [comment]
That's how androids behaves with notifications and LED. It is not controlled by the app.

**Comment by Som30ne** [comment]
Thanks for the update.
Is there a way to set the led to flash again if we are "outsmarting" the OS ?
For example - if the curtain was opened and closed, without snoozing and without dismissing,
create silent notification that will set the led flashing again,
but will not sound any audio, and schedule it to 1 second after the curtain was closed.
(All this - without losing the other required notifications,
 that should be triggered when the notification is not dismissed -
if there are notifications set to sound every XYZ seconds
 until a notification is snoozed or dismissed)

(Hope I explained my self clearly enough)

If there is a way - 
I would like this to be re-opened as an enhancement request.
